lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -1,6 +1,28 @@
 (function() {
+  var self = this;
+  var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
+  function wrapped$() {
+    reporter.increment && reporter.increment(
+      'app_framework.app_scope.violated',
+      1,
+      ['origin:' + self.location.hostname]
+    );
+    return self.$.apply(null, arguments);
+  }
+
+  var wrappedWindow = this.$.extend(true, {}, self, { $: wrapped$ });
```

Comment:
I think you should leave a comment here briefly explaining the purpose of these 3 lines (L13-15)